spec/models/order_detail_spec.rb

```diff
@@ -14,9 +14,9 @@
   before(:each) do
     Settings.order_details.status_change_hooks = nil
     @facility = create(:facility)
-    @facility_account = @facility.facility_accounts.create(attributes_for(:facility_account))
-    @user     = create(:user)
-    @item     = @facility.items.create(attributes_for(:item, facility_account_id: @facility_account.id))
+    @facility_account = create(:facility_account, facility: @facility)
```

Comment:
In some places we use `FactoryBot.create`, and in others we just use `create`, which I’m assuming is available because of [this inclusion](https://github.com/tablexi/nucore-open/blob/d652e997b7c0aaece453926755965cfa9090c1c5/spec/rails_helper.rb#L34).

What do you think about switching these to `FactoryBot.create` so they’re clearer (and so we can maybe ultimately remove that inclusion because `create` is so overloaded in Rails world).

Reply:
I definitely agree with that sentiment. My general rule for this PR was to stay consistent within the each file to keep the number of lines changed smaller (though cleaning up some of the spacing obviously increased that...). I think that's another good thing for a future PR.